lib/cog/models/command.ex

```diff
@@ -50,14 +50,53 @@ defmodule Cog.Models.Command do
     end
   end
 
+  def parse_name(name) do
+    case parse_qualified_name(name) do
+      {:ok, command} ->
+        {:ok, command}
+      {:error, _} ->
+        parse_ambiguous_name(name)
+    end
+  end
+
+  def parse_qualified_name(name) do
+    case String.split(name, ":", parts: 2) do
```

Comment:
I feel like I've seen a function that did this exact thing before. Does that still exist?